build.gradle

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id 'org.springframework.boot' version '2.1.4.RELEASE'
+    id 'java'
+}
+
+apply plugin: 'io.spring.dependency-management'
+
+group = 'pl.allegrotech'
+version = '0.0.1-SNAPSHOT'
```

Comment:
to nie bug to feature ;) tak przynajmniej 0.1.0

Reply:
fixed